### ISSUE TITLE
Use css-only tooltips

### ISSUE THREAD

**Issue by laufhannes** [Runalyze/Runalyze#1617]
**Currently in use:** [bootstrap-tooltip.js](https://github.com/Runalyze/Runalyze/blob/support/2.2.x/lib/bootstrap-tooltip.js)

**My suggestion:** [primercss.io/tooltips/](http://primercss.io/tooltips/) 
This would work without any JS, would not require any event handlers and should be more performant. It works with `content: attr(aria-label)` and pseudo classes `::before` and `::after`.

**Comment by mipapo** [comment]
+1

**Comment by laufhannes** [comment]
- [ ] adjust `Runalyze\View\Tooltip`
 - [ ] change `Ajax::tooltip()`
 - [ ] check for usages of raw html markup for old tooltips
 - [ ] change usages of css classes `atLeft` and `atRight`
 - [ ] remove old css/js
